documentation/prometheus-mixin/alerts.libsonnet

```diff
@@ -317,6 +317,20 @@
               description: '{{ printf "%%.0f" $value }} targets in Prometheus %(prometheusName)s have failed to sync because invalid configuration was supplied.' % $._config,
             },
           },
+          {
+            alert: 'PrometheusQueryOverload',
```

Comment:
Let's perhaps call this `PrometheusHighQueryLoad` then?

Reply:
of course :) will upload new commit soon.